election-ui-kotlin/build.gradle

```diff
@@ -1,24 +1,27 @@
 buildscript {
     ext {
-        androidx_activity_compose_version = '1.5.0'
-        androidx_appcompat_version = '1.4.2'
-        androidx_constraintlayout_version = '2.1.4'
-        androidx_core_version = '1.8.0'
-        androidx_fragment_version = '1.4.1'
-        androidx_lifecycle_version = '2.5.0'
-        androidx_test_espresso_version = '3.4.0'
-        androidx_test_junit_version = '1.1.3'
-        compose_version = '1.1.1'
+        java_version = 11
+        androidx_activity_compose_version = '1.9.3'
+        androidx_activity_version = '1.9.3'
+        androidx_appcompat_version = '1.7.0'
+        androidx_constraintlayout_version = '2.2.0'
+        androidx_core_version = '1.15.0'
+        androidx_fragment_version = '1.8.5'
+        androidx_lifecycle_version = '2.8.7'
+        androidx_test_espresso_version = '3.6.1'
+        androidx_test_junit_version = '1.2.1'
+        compose_version = '1.7.5'
         junit_version = '4.13.2'
-        kotlin_version = '1.6.10'
+        kotlin_version = '1.9.25'
+        kotlin_compiler_version = '1.5.15'
         kotlinx_coroutines_version = '1.6.2'
         kotlinx_serialization_version = '1.3.3'
-        material_version = '1.6.1'
+        material_version = '1.12.0'
     }
 }// Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '8.7.1' apply false
-    id 'com.android.library' version '8.7.1' apply false
+    id 'com.android.application' version '8.7.3' apply false
+    id 'com.android.library' version '8.7.3' apply false
     id 'org.jetbrains.kotlin.android' version "$kotlin_version" apply false
 }
```

Comment:
I'd recommend doing these dependency updates in a separate PR

Reply:
Happy to if we think it'll cause any issues?  I don't know how many people have merge permissions for this repo (I don't), so was just keeping PRs to a minimum.